ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/TheirDetailsMapper.java

```diff
@@ -2,28 +2,25 @@
 
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
-import uk.gov.hmcts.cmc.ccd.domain.CCDDefendant;
 import uk.gov.hmcts.cmc.ccd.domain.CCDPartyType;
+import uk.gov.hmcts.cmc.ccd.domain.defendant.CCDDefendant;
 import uk.gov.hmcts.cmc.ccd.exception.MappingException;
 import uk.gov.hmcts.cmc.domain.models.otherparty.CompanyDetails;
 import uk.gov.hmcts.cmc.domain.models.otherparty.IndividualDetails;
 import uk.gov.hmcts.cmc.domain.models.otherparty.OrganisationDetails;
 import uk.gov.hmcts.cmc.domain.models.otherparty.SoleTraderDetails;
 import uk.gov.hmcts.cmc.domain.models.otherparty.TheirDetails;
 
-import java.time.LocalDate;
-import java.util.Optional;
-
 @Component
```

Comment:
can we keep the name as DefendantMapper rather than TheirDetails?

Reply:
`DefendantMapper` is already there and used as a whole bigger object, and these names are chosen in context with CCD spreadsheet.

Reply:
sorry, just noticed it. the CCD spreadsheet doesnt have theirdetails as type at all. Let we try renaming it in the next PR you were doing for refactoring